project-docs/wave-03.md

```diff
@@ -1,24 +1,60 @@
 # Wave 03:  useEffect And Axios
 
-Next we will add the [`axios`](https://github.com/axios/axios) library to our project.  We will use this library to make a request to our API to get the list of tasks.
+In Wave 03 we will add the [`axios`](https://github.com/axios/axios) library to our project.  We will use this library to make a request to our API to get the list of tasks.
 
 When the user deletes or updates a task we will make calls to the API to update the list of tasks.
 
-### API EndPoints:
+## Make Axios requests in `App.js`
 
-The API is active on Heroku at [`https://adas-task-list.herokuapp.com`](https://adas-task-list.herokuapp.com).
+Create or refactor the following features to connect the Task List Front End to the Task List Back End (Task List API)
+
+- In the `App`, use the `useEffect` hook to make an API call to get the list of tasks from the database when the React app is loaded.
+- Update the `toggleCompleteTask` callback function (the function that toggles the state of completion; it may have different name) in the `App` such that marking a task complete or incomplete in the front end makes a change to the state of the task in the database. 
```

Comment:
I could see some students, particularly students who are still working on grasping these topics, not making the connection back to what they did in the previous wave and therefore being confused about why the function 'may have a different name'
```suggestion
- Update the `toggleCompleteTask` callback function (the function that toggles the state of completion; you may have named it something different) in the `App` such that marking a task complete or incomplete in the front end makes a change to the state of the task in the database. 
```